build.sbt

```diff
@@ -205,7 +235,9 @@ def circeModule(path: String, mima: Option[String]): Project = {
   val id = path.split("-").reduce(_ + _.capitalize)
   Project(id, file(s"modules/$path"))
     .configure(circeProject(path))
-    .settings(mimaPreviousArtifacts := mima.map("io.circe" %% moduleName.value % _).toSet)
+    .settings(
+      mima.map(version => mimaPreviousArtifacts := Set("io.circe" %%% moduleName.value % version)).toSeq
```

Comment:
I'm afraid this is not sufficient. You actually need to compare against _all_ previous releases in the current series. See https://github.com/circe/circe/pull/1906#discussion_r788919550.

Reply:
👍, sorry, noticed your comment on the other PR after I had already pushed this up.  I'll change it up so it does a more though test of the versions when I get a chance.

Reply:
I'm a broken record, but you should consider using [sbt-typelevel](https://github.com/typelevel/sbt-typelevel/) or at least sbt-typelevel-ci-release. It determines previous MiMa artifacts automatically. It'll also automatically configure your build to cross on JVM/JS like you've done manually here and also fixes the bug that forced you to set `githubWorkflowArtifactUpload := false`.

Reply:
I'll consider switching after we get a release out... right now I'm just trying to get some basic things into CI, not ready to rock the boat by switching the publishing methods and other stuff just yet.